.github/workflows/bson-ruby.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: >-
+      ${{ matrix.os }} ${{ matrix.ruby }}
+    env:
+      CI: true
+      TESTOPTS: -v
+
+    runs-on: ${{ matrix.os }}-latest
+    if: |
+      !(   contains(github.event.pull_request.title,  '[ci skip]')
+        || contains(github.event.pull_request.title,  '[skip ci]')
+        || contains(github.event.head_commit.message, '[ci skip]')
+        || contains(github.event.head_commit.message, '[skip ci]'))
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu, macos, windows ]
+        ruby: [ 2.3, 2.4, 2.5, 2.6, 2.7, head ]
+        include:
+          - { os: windows , ruby: mingw }
+        exclude:
+          - { os: windows , ruby: head }
```

Comment:
Also why is windows/ruby head excluded? Is this not a possible configuration?

Reply:
@p-mongo

Sorry for the delay, having issues with internet service.

If you want to stick with Travis & AppVeyor, that's fine.  There are pro's and con's to either.  Actions has a much higher limit for concurrent jobs.  One con re Actions is that it has no 32-bit Ruby support on Windows.

As to ruby-head vs ruby-mingw, they're both master builds.  I am the builder/supplier of the mingw and mswin builds.  They're fully tested and used by several repos, including ruby organization repos.  Your choice.